mkdocs.yml

```diff
@@ -192,6 +193,7 @@ nav:
               - Ticket store: docs/reference/modules/Users/TicketStore.md
           - Home Route: docs/reference/modules/HomeRoute/README.md
           - Feeds: docs/reference/modules/Feeds/README.md
+          - Commerce: https://commerce.orchardcore.net/en/latest
```

Comment:
OCC is kind of special because it's the only project that's officially supported (i.e. under https://github.com/OrchardCMS) but not part of OC itself. I'm OK with having it like this if it's apparent that it's not part of OC proper (which needs to be handled on https://commerce.orchardcore.net).